### PR TITLE
'LIon' -> 'Lion' everywhere

### PR DESCRIPTION
**Pull request by dslmeinte**
+ add CHANGELOG

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,17 +56,17 @@ The following is a list of links to potential starting points:
 
 * [Diagrams](diagrams/) - various generated diagrams.
   The PlantUML file [`diagrams/metametamodel-gen.puml`](diagrams/metametamodel-gen.puml) is generated from the meta-circular [self-definition of `lioncore`](src-pkg/m3/lioncore.ts).
-  This generated PlantUML file can then be compared with [this one](https://github.com/LIonWeb-org/organization/blob/main/metametamodel/metametamodel.puml): they should have exactly the same contents apart from a couple of obvious differences.
+  This generated PlantUML file can then be compared with [this one](https://github.com/LionWeb-org/organization/blob/main/metametamodel/metametamodel.puml): they should have exactly the same contents apart from a couple of obvious differences.
```
Comment (enikao) [inline README.md:59]: Can we rename the organization?

File: models/README.md [hunk 2]
```diff
@@ -6,13 +6,13 @@ A split is made between [languages](./meta), and [instances](./instance) of thos
 
 ## Languages
 
-Serializations of languages, i.e. instances of the `Language` (meta-)concept from LIonCore/M3.
+Serializations of languages, i.e. instances of the `Language` (meta-)concept from LionCore/M3.
```
Comment (enikao) [inline models/README.md:9]: Is `LionCore/M3` (with a slash) really the correct spelling?

File: schemas/library.serialization.schema.json [hunk 3]
```diff
@@ -76,7 +76,7 @@
         },
         "concept": {
           "const": {
-            "metamodel": "LIonCore_M3",
+            "metamodel": "LionCore_M3",
```
Comment (enikao) [inline schemas/library.serialization.schema.json:79]: `LionCore_M3` seems outdated?

File: schemas/lioncore.m3.serialization.schema.json [hunk 4]
```diff
@@ -76,7 +76,7 @@
         },
         "concept": {
           "const": {
-            "metamodel": "LIonCore_M3",
+            "metamodel": "LionCore_M3",
```
Comment (enikao) [inline schemas/lioncore.m3.serialization.schema.json:79]: Outdated?

File: src-pkg/api.ts [hunk 5]
```diff
@@ -14,7 +14,7 @@ type ConceptDeducer<NT extends Node> = (node: NT) => Concept
  * Two interfaces that defines APIs for in-memory models.
  * Instances/implementations of this interface parametrize generic (de-)serialization.
  * Implementations of ModelAPI {w|c}ould be:
- *  - specific to LIoncore (so to match m3/types.ts)
+ *  - specific to Lioncore (so to match m3/types.ts)
```
Comment (enikao) [inline src-pkg/api.ts:17]: `LionCore`?

File: README.md (hunk 1 above)
Comment (dslmeinte) [reply README.md:59]: Looks like it: https://docs.github.com/en/organizations/managing-organization-settings/renaming-an-organization

File: schemas/library.serialization.schema.json (hunk 3 above)
Comment (dslmeinte) [reply schemas/library.serialization.schema.json:79]: If you don't mind, we go over the precise IDs and keys in a separate PR.

File: schemas/lioncore.m3.serialization.schema.json (hunk 4 above)
Comment (dslmeinte) [reply schemas/lioncore.m3.serialization.schema.json:79]: If you don't mind, we go over the precise IDs and keys in a separate PR.

File: models/README.md (hunk 2 above)
Comment (enikao) [reply models/README.md:9]: I didn't read the context, just wanted to make sure it's not a missed instance. I don't have a preference.

**Review by woppa684 — APPROVED** [review]
Apart from the comments from Niko, the changes look good to me. I guess this has been agreed? So that the MPS implementation will also start sending the keys with a lowercase i soon? @enikao

**Comment by enikao** [comment]
> So that the MPS implementation will also start sending the keys with a lowercase i soon? @enikao

Yes, working on it.

**Pull request merged**